### ISSUE TITLE
VAD not recognizing foreign language as speech

### ISSUE THREAD

**Issue by zoha21** [marsbroshok/VAD-python#4]
Hi. Is there a reason why VAD would fail to identify speech in foreign language in some cases? I tested it using some audio in spanish and french, and the convert_windows_to_readible_labels() failed to return any labels for those audio. I was wondering if there were some parameters I needed to initialize in those cases

**Comment by Sadam1195** [comment]
No, language is not the issue. I think there is some logical error in code (idk where and what) which is why it doesn't return correct timestamps of speech detected parts in the audio file. I have checked against multiple files.
